src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_HUEditor_Launcher.java

```diff
@@ -78,20 +78,22 @@ protected ProcessPreconditionsResolution checkPreconditionsApplicable()
 		}
 
 		final PPOrderLinesView ppOrderLineView = getView();
-		if (!(ppOrderLineView.isStatusPlanning() || ppOrderLineView.isStatusReview()))
+		final PPOrderLineRow singleSelectedRow = getSingleSelectedRow();
+
+		final boolean isIssueOnlyWhatWasReveiced = X_PP_Order_BOMLine.ISSUEMETHOD_IssueOnlyForReceived.equals(singleSelectedRow.getIssueMethod());
+
+		if (!(ppOrderLineView.isStatusPlanning() || ppOrderLineView.isStatusReview()) && !isIssueOnlyWhatWasReveiced)
 		{
 			final String internalReason = StringUtils.formatMessage("The PP_Order of the the current ppOrderLineView is not in planning or in review; ppOrderLineView={}", ppOrderLineView);
 			return ProcessPreconditionsResolution.rejectWithInternalReason(internalReason);
 		}
 
-		final PPOrderLineRow singleSelectedRow = getSingleSelectedRow();
-
 		if (!singleSelectedRow.isIssue())
 		{
 			final String internalReason = StringUtils.formatMessage("The selected ppOrderLineRow is not an issue row; selectedRow={}", singleSelectedRow);
 			return ProcessPreconditionsResolution.rejectWithInternalReason(internalReason);
 		}
-		if (singleSelectedRow.isProcessed() && !(X_PP_Order_BOMLine.ISSUEMETHOD_IssueOnlyForReceived.equals(singleSelectedRow.getIssueMethod())))
+		if (singleSelectedRow.isProcessed() && !isIssueOnlyWhatWasReveiced)
```

Comment:
> also, the reject resaon now doesn't reflect the expression anymore

same here (ok, aparently it didn't reflect the reason to start with)

generally, please explain what role `isIssueOnlyWhatWasReveiced` is playing in here to begin with (bc I for my part think I don't understand it)

Reply:
Done